### PR TITLE
BETA: Register subyyal.is-a.dev

### PR DESCRIPTION
**Pull request by Subyyalx**
Added `subyyal.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Review by triach-rold — APPROVED** [review]
Website is reachable and seems to be complete enough.

**Review by Subyyalx — COMMENTED** [review]
Looks good

File: domains/subyyal.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Subyyalx",
+        "email": "subyyal@xavor.com"
+    },
+    "record": {
+        "CNAME": "https://infraswift.itactivities.com"
```
Comment (CuteDog5695) [inline domains/subyyal.json:7]: Renove https:// from the CNAME record

**Review by iostpa — DISMISSED** [review]

**Review by Subyyalx — COMMENTED** [review]
done

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Review by Subyyalx — COMMENTED** [review]
removed "

**Review by Subyyalx — COMMENTED** [review]
done

**Review by triach-rold — APPROVED** [review]

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**